src/components/VideoPlayer/VideoInformation/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useContext } from 'react';
+import Grid from '@material-ui/core/Grid';
+import AddToFavorites from '../AddToFavorites';
+import { store } from '../../../state/store';
+
+import { Title, Description } from './style';
+import Profile from '../Profile';
+
+const VideoInformation = () => {
+  const globalState = useContext(store);
+  const title =
+    globalState.state.currentVideo.snippet === undefined
```

Comment:
Review that the currentVideo is being changed whenever you click in a new video card. I saw a bug in which you can click a link on the suggested videos and neither the information nor the suggested video list will change 
<img width="1785" alt="Screen Shot 2021-04-05 at 15 28 47" src="https://user-images.githubusercontent.com/16677116/113623323-d23a7400-9623-11eb-9e2a-f33d506a72ca.png">
<img width="1785" alt="Screen Shot 2021-04-05 at 15 28 54" src="https://user-images.githubusercontent.com/16677116/113623359-dcf50900-9623-11eb-8315-8fc03b710cff.png">